contrib/python-bindings/source/export_triangulation.cc

```diff
@@ -80,70 +102,227 @@ namespace python
 
 
 
+  const char generate_subdivided_steps_hyper_rectangle_docstring [] =
```

Comment:
damn, looks like one still needs to manually copy-paste here....

Reply:
Yeah this is the python documentation. It is totally independent of doxygen.